webapp/src/Migrations/Version20201219154651.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20201219154651 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
```

Comment:
Can we/should we have descriptions? I don't know if we do elsewhere in the migrations...

Reply:
I think most/all have them yes

Reply:
Done.